Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -356,7 +356,7 @@ extension StreamOrchestrator: SubscriptionManagerDelegate {
     }
 
     public func onStatsReport(_ report: MCStatsReport) {
-        guard let streamingStats = StreamingStatistics(report) else {
+        guard let streamingStats = AllStreamingStatistics(report) else {
```

Comment:
I don't think SDK can change this, since `report` object is only accessible inside the callback. When the callback is done, the reference can be destroyed (and that what's happening during crash). 
I reckon we shouldn't get rid of `AllStreamingStatistics`.

Reply:
Sure, we can temporarily use the current implementation, can we add a `TODO: comment` on what's the current limitations. It needs to be fixed. 

SDK callbacks cannot force the user to use the passed in arguments in the same thread. These days with ARC, objects are not explicitly released. When passed around (say into another closure or another asynchronous task) there would still be a non-zero reference count and the object will not be released unless the SDK author has explicitly chosen to set the object to `nil`.